website/docs/quickstart/flink.md

```diff
@@ -33,7 +33,17 @@ mkdir fluss-quickstart-flink
 cd fluss-quickstart-flink
 ```
 
-2. Create a `docker-compose.yml` file with the following content:
+2. Create a Dockerfile named `fluss.Dockerfile` as follows. You can adjust the Flink version as needed. Please make sure to download the compatible versions of [fluss-flink connector jar](/downloads), [flink-connector-faker](https://github.com/knaufk/flink-faker/releases) in the Dockerfile. 
+
+```Dockerfile
```

Comment:
A docker file looks wried to me.  Could we just mount the libs to flink image just like we did for iceberg?